components/aws/sources/redshift-updated-row/redshift-updated-row.mjs

```
@@ -0,0 +1,99 @@
+      workgroupName,
+      database,
+      sql,
+      parameters,
+    });
+
+    if (response?.Records?.length > 0) {
+      const columnNames = response.ColumnMetadata.map((c) => c.name);
+
+      const records = response.Records.map((record) =>
+        record.reduce((row, field, index) => ({
+          ...row,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)